applications/nrf5340_audio/src/bluetooth/bt_content_control/bt_content_ctrl.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BT_CONTENT_CTRL_H_
+#define _BT_CONTENT_CTRL_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+/**
+ * @brief	Start content.
```

Comment:
That depends on the content. It could be media, it could be a call etc. I deliberately made it vague since this is just the shim that will cover all the different content control services.

Reply:
Would my suggestion work here then?

Reply:
No, since we are not starting the service, the service is just there and you use the service to start the content

Reply:
```suggestion
 * @brief	Send the start request for content transmission.
```
And similarly for stop, as discussed.